common/rabbitmq/values.yaml

```diff
@@ -112,3 +112,7 @@ upgrades:
 #  rollingUpdate:
 #    maxUnavailable: 1
 #    maxSurge: 3
+
+linkerd:
+  #linkerd annotation for the RabbitMQ pod (true/false)
```

Comment:
nit: missing a space after the `#` and annmotations also happen for Service objects not only for pods

Reply:
space added after `#`